pages/learn/develop/container-contracts.md

```diff
@@ -44,7 +44,7 @@ requires:
 
 __Note__ If `name`, `type`, or `slug` are omitted, the build with fail.
 
-Multiple requirements may be added to a single contract. The following example requires that the hostOS is greater than or equal to 2.44.0, and the Supervisor version is greater than 10.
+Multiple requirements may be added to a single contract. The following example requires that the {{ $names.os.lower }} is greater than or equal to 2.44.0, and the Supervisor version is greater than 10.
```

Comment:
I think we will have a problem with ESR os versions here.

Reply:
I think the issue here is the type `sw.os` is too generic. We often said ESR should be considered a different OS, as such it should have a different contract. In general we might have several os contracts (ubuntu, raspbian, ecc) in the system. The contract should express that it really wants balenaOS, so something like.
```
requires:
  - type: 'sw.os'
    version: '>=2.44.0'  
    slug: 'balenaOS'
```

This still sounds tricky to get right with ESR though, as we might want to represent the idea that compatibility with certain balenaOS versions imply compatibility with certain ESR versions. I think the correct way to do this is to further decompose the OS contract in a set of capabilities which are independent of the OS slug